packages/vite/src/node/plugins/resolve.ts

```diff
@@ -416,23 +453,28 @@ export function tryNodeResolve(
     basedir = root
   }
 
+  const preserveSymlinks = !!server?.config.resolve.preserveSymlinks
+
   // nested node module, step-by-step resolve to the basedir of the nestedPath
   if (nestedRoot) {
     basedir = nestedResolveFrom(nestedRoot, basedir)
```

Comment:
In #4634, I added a new `nestedResolveFrom` function which uses `resolveFrom` under-the-hood. We might want to update that function too:

https://github.com/vitejs/vite/blob/284170edc29c7556e04491450e0db3b71a14ec08/packages/vite/src/node/utils.ts#L71

Reply:
Done, thank you for the reminder.